Dockerfile_dev

```diff
@@ -0,0 +1,61 @@
+FROM jupyter/base-notebook:eb70bcf1a292
+USER root
+
+ARG hnnuiBranch=skeleton
```

Comment:
@afonsobspinto change this to development
```suggestion
ARG hnnuiBranch=development
```

Reply:
👍